docs/config/lua/window/get_appearance.md

```diff
@@ -45,4 +45,40 @@ end)
 return {
 }
 ```
+### Wayland GNOME Workaround
```

Comment:
I'd prefer to phrase this section something like:

```
### Wayland GNOME Appearance

Since Wayland doesn't specifically support appearance, we can use this
alternative method to determine the appearance and adapt the color
scheme accordingly.
```

Reply:
All set in latest commit.